OpenDreamServer/Server.cs

```diff
@@ -14,6 +16,8 @@ class Connection : DreamConnection
         private readonly NetworkStream _tcpStream;
         private readonly BinaryReader _tcpStreamBinaryReader;
         private readonly BinaryWriter _tcpStreamBinaryWriter;
+        
+        public ClientData ClientData;
```

Comment:
`ClientData` is already in the inherited class `DreamConnection`
```suggestion
```

Reply:
yeah it hid the field, no idea how i missed that